src/openrct2/world/Footpath.cpp

```diff
@@ -1049,34 +1045,32 @@ void footpath_connect_edges(int32_t x, int32_t y, TileElement* tileElement, int3
  *  rct2: 0x006A742F
  */
 void footpath_chain_ride_queue(
```

Comment:
The changes you have made will break this function. It might be easier to start again. It needs to be reworked this function as its a confusing mess. I've made some suggestions but they need to be thought through a bit more.